core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -241,6 +287,44 @@ export class WorkflowEditorComponent implements AfterViewInit {
     return jointPaperOptions;
   }
 }
+/**/
+function getObservables(domItem: any) {
+  const mouseEventToCoordinate = (mouseEvent: any) => {
+    mouseEvent.preventDefault();
+    return {
+      x: mouseEvent.clientX,
+      y: mouseEvent.clientY
+    };
+  };
+
+  const touchEventToCoordinate = (touchEvent: any) => {
+    touchEvent.preventDefault();
+    return {
+      x: touchEvent.changedTouches[0].clientX,
+      y: touchEvent.changedTouches[0].clientY
+    };
+  };
+
+  const wheelEventToCoordinate = (WheelEvent: any) => {
+    WheelEvent.preventDefault();
+    return {
+      x: WheelEvent.changedTouches[0].clientX,
+      y: WheelEvent.changedTouches[0].clientY
+    };
+  };
+  const mouseDowns = Observable.fromEvent(domItem, 'mousedown').map(mouseEventToCoordinate);
+  const mouseMoves = Observable.fromEvent(window, 'mousemove').map(mouseEventToCoordinate);
+  const mouseUps = Observable.fromEvent(window, 'mouseup').map(mouseEventToCoordinate);
+
+  const wheelCall = Observable.fromEvent(domItem, 'mousewheel').map(wheelEventToCoordinate);
+
+  const touchStarts = Observable.fromEvent(domItem, 'touchstart').map(touchEventToCoordinate);
+  const touchMoves = Observable.fromEvent(domItem, 'touchmove').map(touchEventToCoordinate);
+  const touchEnds = Observable.fromEvent(window, 'touchend').map(touchEventToCoordinate);
+
+  // s
+  return { mouseDowns, mouseMoves, mouseUps, wheelCall };
```

Comment:
With a correct use of `Observables`, I don't think this method is necessary. Please add some comments about what this method do specifically.